components/microsoft_teams/microsoft_teams.app.mjs

```diff
@@ -58,76 +59,77 @@ export default {
     chat: {
       type: "string",
       label: "Chat",
-      description: "Team Chat (internal and external contacts)",
-      async options({ prevContext }) {
-        const response = prevContext.nextLink
+      description: "Select a chat (type to search by participant names)",
+      async options({
+        prevContext, searchTerm,
```

Comment:
Does `searchTerm` work as expected here? I thought the parameter was named `query`

Reply:
Thanks - updated to `query` here.